### PR TITLE
Update Jakarta Data 1.0.0-b2

### PR DESCRIPTION
**Pull request by otaviojava**
# Context

When creating a specification project release review, create PRs with the content defined as follows.

Include the following in the PR:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [x] Specification PDF in the form of wombat-spec-x.y.pdf
- [x] Specification HTML in the form of wombat-spec-x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md

### CODE REVIEW

File: data/1.0/_index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+### Removals, deprecations or backwards incompatible changes
+<!-- List here -->
+* N/A 
+
+### Minimum Java SE Version
+<!-- Specify the minimum required Java SE version for this specification -->
+**Java SE 11 or higher**
+
+
+* [Jakarta Data Release Plan](https://projects.eclipse.org/projects/ee4j.data/governance)
```
Comment (ivargrimstad) [inline data/1.0/_index.md:49]: `[Jakarta Data Release Record](https://projects.eclipse.org/projects/ee4j.data/releases/1.0)`

File: data/1.0/_index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+* [Jakarta NoSQL 1.0 Specification Document](./data-1.0.0-b2.pdf) (PDF)
+* [Jakarta NoSQL 1.0 Specification Document](./data-1.0.0-b2.html) (HTML)
+* [Jakarta NoSQL 1.0 Specification Javadoc](./apidocs)
+* Maven coordinates
+  * [jakarta-data-api:jakarta.data:jar:1.0.0-b2](https://repo1.maven.org/maven2/jakarta/data/jakarta-data-api/1.0.0-b2/)
+
+
+
+# Compatible Implementations
```
Comment (ivargrimstad) [inline data/1.0/_index.md:58]: I don't think we should list these before there is a TCK available.

**Review by ivargrimstad — CHANGES_REQUESTED** [review]
A couple of changes were suggested. 
When these are addressed, I can start the plan review.

File: data/1.0/_index.md (hunk 1 above)
Comment (otaviojava) [reply data/1.0/_index.md]: Thanks, updated.

**Review by ivargrimstad — APPROVED** [review]

**Pull request merged**